src/plugins/cordova-plugin-network-information/sim-host.js

```diff
@@ -30,6 +37,18 @@ function initialize() {
     connectionTypeList.addEventListener('change', function () {
         network.updateNetworkType(connectionTypeList.value);
     });
+
+    registerTelemetryEvents();
+}
+
+function registerTelemetryEvents() {
+    var connectionList = document.getElementById('connection-list');
+
+    connectionList.onchange = function () {
+        var option = connectionList.options[connectionList.selectedIndex];
+
+        telemetry.sendUITelemetry(Object.assign({}, baseProps, { control: 'connection-list', value: option.value }));
+    };
```

Comment:
There is also a telemetry event to indicate which plugins are being run with the app. The event reports 3 different list of plugins that the simulated project is using:

- `simulatedBuiltIn`: the list of default cordova-simulate plugins
-`simulatedNonBuiltIn`: the list of non-default plugins for which there was an implementation of the simulation layer
-`notSimulated`: the list of plugins that don't have an implemented simulation

With these 3 lists, we should be able to answer your question about adding new plugins!